src/components/messenger/list/conversation-list-panel.tsx

```diff
@@ -9,24 +9,40 @@ import { IconButton } from '../../icon-button';
 import { ConversationItem } from './conversation-item';
 import { InviteDialogContainer } from '../../invite-dialog/container';
 import { Button, Modal } from '@zero-tech/zui/components';
+import { Item, Option } from '../lib/types';
+import { UserSearchResults } from './user-search-results';
+import { itemToOption } from '../lib/utils';
 
 export interface Properties {
   conversations: Channel[];
 
-  onConversationClick: (conversationId: string) => void;
+  search: (input: string) => any;
   startConversation: () => void;
+  onCreateConversation: (userId: string) => void;
+  onConversationClick: (conversationId: string) => void;
 }
 
 interface State {
   filter: string;
   inviteDialogOpen: boolean;
+  userSearchResults: Option[];
 }
 
 export class ConversationListPanel extends React.Component<Properties, State> {
-  state = { filter: '', inviteDialogOpen: false };
+  state = { filter: '', inviteDialogOpen: false, userSearchResults: [] };
 
-  searchChanged = (search: string) => {
+  searchChanged = async (search: string) => {
     this.setState({ filter: search });
+
+    if (!search) {
+      return this.setState({ userSearchResults: null });
+    }
+
+    const items: Item[] = await this.props.search(search);
+    const conversationMemberIds = this.props.conversations.flatMap((c) => c.otherMembers.map((m) => m.profileId));
+    const filteredItems = items?.filter((item) => !conversationMemberIds.includes(item.id));
```

Comment:
:thinking: I think there might be a usability question here. If I'm in a group conversation with you and someone else.... and then I try to start a one on one conversation with only you....you won't show up in the results, right? I think we want to show existing convos with a match but only filter out the one on one based ones from the api search results?

Reply:
When I create a group and then search one of those users that I do not have a current conversation with they show up in the search results, I think thats what you mean ? 😬 

https://github.com/zer0-os/zOS/assets/39112648/21509fe9-0001-40e0-98e6-1d505d749352